src/main/java/au/gov/ga/geodesy/igssitelog/support/marshalling/moxy/DateAdapter.java

```diff
@@ -1,72 +1,40 @@
 package au.gov.ga.geodesy.igssitelog.support.marshalling.moxy;
 
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-
-import java.util.Arrays;
-import java.util.Date;
-import java.util.List;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
 
 import javax.xml.bind.annotation.adapters.XmlAdapter;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class DateAdapter extends XmlAdapter<String, Date> {
-
-    private static final Logger log = LoggerFactory.getLogger(DateAdapter.class);
-
-    // the intention is to be lenient when parsing dates, feel free to add more patterns
-    private static String[] inputDatePatterns = new String[] {"yyyy-MM-dd", "yyyy", "(yyyy-MM-dd)"};
-
-    private static String[] nullDatePatterns = new String[] {
-            "(CCYY-MM-DDThh:mmZ)",
-            "CCYY-MM-DDThh:mmZ",
-            "(CCYY-MM-DD)",
-            "CCYY-MM-DD"
-    };
```

Comment:
I think it is useful knowing why we return null, a) because we recognized a null pattern, or b) because we failed to parse a date string. Today we return null in both cases, but one day we may become more strict and fail on bad date strings.